extra/linux/org.alacritty.Alacritty.appdata.xml

```diff
@@ -1,34 +1,45 @@
+<!-- SPDX-License-Identifier: MIT -->
```

Comment:
It looks like that the SPDX header needs to be below the xml doctype declaration:

```
$ appstreamcli validate org.alacritty.Alacritty.appdata.xml
E: ~:~: xml-markup-invalid
     Could not parse XML data: Entity: line 2: parser error : XML declaration allowed only at
     the start of the document
     <?xml version="1.0" encoding="UTF-8"?>
          ^
     

✘ Validation failed: errors: 1
```